wp-resources/themes/webonary-zeedisplay/footer.php

```diff
@@ -1,146 +1,145 @@
+			    	<a href="<?php echo $link; ?>" class="responsive-menu-item-link">
+			            <?php echo $title; ?>
+			            <?php
+			            if($next_item_parent != 0)
+			            {
+			            ?>
+			            <div class="responsive-menu-subarrow">▼</div>
+			            <?php
+			            }
+			            ?>
+			         </a>
+		        <?php
+			}
+		        if ( $parent_id == $item->menu_item_parent )
+		        {
+		        ?>
+		            <?php
+		            if ( !$submenu )
+		            {
+		            	$submenu = true; ?>
+		            	<ul class="responsive-menu-submenu responsive-menu-submenu-depth-1">
+		            <?php
+		            } ?>
+
+		                <li id="responsive-menu-item-<?php echo $item->ID; ?>" class="menu-item menu-item-type-post_type menu-item-object-page responsive-menu-item">
+		                <a href="<?php echo $link; ?>"
+								class="responsive-menu-item-link"><?php echo $title; ?></a></li>
+
+		            <?php if ( $next_item_parent != $parent_id && $submenu )
+		            {
+		            ?>
+		            </ul>
+		            <?php $submenu = false;
+		            }
+		            ?>
+
+		        <?php
+		        }
+
+		    if ( $next_item_parent != $parent_id )
+		    {
+		    ?>
+		    </li>
+		    <?php $submenu = false;
+		    }
+		    ?>
+
+		<?php
+		}
+		?>
+		</ul>
+
+		<div id="responsive-menu-search-box">
+			<form action="/"
+				class="responsive-menu-search-form" role="search">
+				<input name="s" placeholder="Search"
+					class="responsive-menu-search-box" type="search">
+			</form>
+		</div>
+		<div id="responsive-menu-additional-content"></div>
+	</div>
+</div>
+<?php wp_footer();?>
+		<?php
+		$options = get_option('themezee_options');
+		if(is_active_sidebar('sidebar-footer')) : ?>
+		<div id="bottombar">
+			<ul>
+				<?php dynamic_sidebar('sidebar-footer'); ?>
+			</ul>
+			<div style="clear: both;"></div>
+		</div>
+		<?php endif; ?>
+
+		<?php
+
+		if (has_nav_menu('top_navi'))
+		{
+			$color="pink";
+		}
+		else
+		{
+			$color="blue";
+		}
+		?>
+		<div id="footer" class="<?php echo $color; ?>" style="text-align:center; padding-bottom: 8px; font-size:12px;">
+			<div style="text-align:center;"><?php echo str_replace("[year]", date("Y"),$options['themeZee_footer']); ?></div>
+			<hr style="font-size:5px; margin-bottom: 4px; clear:both;">
+			<div id="copyright">
+				<div class="cr-left"><img src="<?php echo  get_template_directory_uri(); ?>/images/sil-icon.gif" style="vertical-align:middle;"> <span style="margin-left:10px;">© <?php echo "2013 - " . date("Y"); ?> <a href="https://www.sil.org" target="_blank">SIL International</a><sup>®</sup></span></div>
+				<div class="cr-center"><img src="<?php echo  get_template_directory_uri(); ?>/images/webonary-icon.png" style="vertical-align:middle;"> <span><a href="https://www.webonary.org" target="_blank">Webonary.org</a></span></div>
```

Comment:
```suggestion
				<div class="cr-center"><img src="<?php echo  get_template_directory_uri(); ?>/images/webonary-icon.png" style="vertical-align:middle;"> <span><a href="<?php echo get_site_url()?>" target="_blank">Webonary.org</a></span></div>
```

Reply:
The changes you are recommending may be appropriate, but that decision is not for me to make. Also, I think they should be in a new Issue since they aren't related to this task.